src/views/user/LogEntryCardsView/LogEntry.js

```diff
@@ -0,0 +1,181 @@
+  },
+  imageSizing: {
+    width: '5%',
+    height: '5%',
+    padding: '0.25rem'
+  },
+  expand: {
+    transform: 'rotate(0deg)',
+    marginLeft: 'auto',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest
+    })
+  },
+  expandOpen: {
+    transform: 'rotate(180deg)'
+  }
+}));
+
+// ----------------------------------------------------------------------
+
+LogEntry.propTypes = {
+  //   log: PropTypes.object.isRequired,
+  className: PropTypes.string
+};
+
+function LogEntry({ className }) {
+  const classes = useStyles();
+  const [expanded, setExpanded] = React.useState(false);
+  const [newLog, setNewLog] = React.useState('');
+  const [logEntries, setLogEntries] = React.useState([]);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  const handleSubmit = () => {
+    setLogEntries(logEntries.concat(newLog));
+    setNewLog('');
+  };
+
+  const date = new Date().toLocaleTimeString('en-US', {
+    hour12: false,
+    hour: 'numeric',
+    minute: 'numeric'
+  });
+  return (
+    <CardActionArea>
+      <Card
+        className={clsx(classes.root, className)}
+        display="flex"
+        justifyContent="between"
+      >
+        <CardHeader
+          action={
+            <IconButton aria-label="settings">
+              <MoreVertIcon />
+            </IconButton>
+          }
+        />
+        <div className={classes.cardMediaWrap}>
+          <Avatar
+            variant="square"
+            alt={'log-entry'}
+            src={'https://image.flaticon.com/icons/png/512/2648/2648958.png'}
+            sx={{
+              width: 70,
+              height: 70,
+              zIndex: 11,
+              position: 'absolute',
+              transform: 'translateY(-50%)'
+            }}
+          />
+        </div>
+
+        <CardContent className={classes.cardContent}>
+          <Box display="flex" justifyContent="center" width={1}>
+            <Rating
+              defaultValue={0}
+              precision={0.5}
+              max={4}
+              emptyIcon={<FiberManualRecordIcon />}
+              icon={<FiberManualRecordIcon />}
+            />
+          </Box>
+          <Typography variant="h6" align="center">
+            {date}
+          </Typography>
+          <Box textAlign="center">
+            <IconButton
+              className={clsx(classes.expand, {
+                [classes.expandOpen]: expanded
+              })}
+              onClick={handleExpandClick}
+              aria-expanded={expanded}
+              aria-label="show more"
+            >
+              <ExpandMoreIcon />
+            </IconButton>
+          </Box>
+        </CardContent>
+        <Collapse in={expanded} timeout="auto" unmountOnExit>
+          <CardContent>
+            <Typography paragraph>
+              {logEntries.map(logEntry => (
+                <List width={1} sx={{ p: 0 }}>
+                  <Editable logEntry={logEntry} />
```

Comment:
As defined in the Acceptance Criteria section for https://code-sherpas.jetbrains.space/p/katydid/checklists/1sROls1LA95L, Adding the comments section on each log entry was out of the scope of this User story. because the backend is not ready for creating, updating, deleting comments. If we merge this, we would be showing to our end user, a feature that is not fully supported yet.

From the US Acceptance Criteria description:
```
Scenario: Card is not expanded when I click on the expand arrow
  Given: There is a card with an 'expand' icon
  When: I click on the arrow
  Then: Nothing happens


```